cmd/subnetcmd/deploy.go

```diff
@@ -196,9 +196,6 @@ func deploySubnet(_ *cobra.Command, args []string) error {
 	case models.SpacesVM:
 		var genesis spacesvmchain.Genesis
 		err = json.Unmarshal(chainGenesis, &genesis)
-	default:
```

Comment:
Could we think of some other sensible `default`? Or should we have no default case whatsoever.

Reply:
Because a custom VM can arbitrarily define the genesis format, there is no generalized verification that can be performed here.